src/components/DacVotePanel.js

```diff
@@ -139,28 +167,51 @@ export const DacVotePanel = hh(class DacVotePanel extends React.PureComponent {
     }
   };
 
+  // checks if required fields are completed before posting votes
+  submitChairVote = () => {
+    const { finalVote } = this.state;
+    if (finalVote.vote !== null) {
+      this.submitVote(finalVote);
+      this.closeElection();
+    } else {
+      console.log('Please complete all required fields');
+    };
+  };
+
   // posts the supplied vote for this DAR
   async submitVote(vote) {
     const { darId } = this.props.ids;
-    if (vote.createDate === null) {
-      try {
+    try {
+      if (vote.type === 'FINAL') {
+        await Votes.updateFinalAccessDarVote(darId, vote);
+        console.log('Final vote sumbitted');
+      } else if (vote.createDate === null) {
         await Votes.postDarVote(darId, vote);
         console.log('Vote sumbitted');
-      }
-      catch (e) {
-        console.log('Something went wrong. Try again.')
-      };
-    } else {
-      try {
+      } else {
         await Votes.updateDarVote(darId, vote);
         console.log('Vote edited');
       }
-      catch (e) {
-        console.log('Something went wrong. Try again.')
-      };
     }
+    catch (e) {
+      console.log('Something went wrong. Try again.')
+    };
   };
 
+  // closes the election for this DAR
+  async closeElection() {
+    const { election } = this.props;
+    const electionClone = _.cloneDeep(election);
+    electionClone.status = 'Closed';
+    try {
+      await Election.updateElection(election.electionId, electionClone);
+      console.log('Election closed');
```

Comment:
Not sure we need to capture this as something to show the user ... I would leave this out. Keep the error logging and we'll deal with that in the error handling story.